packages/replay/src/install.ts

```diff
@@ -114,13 +129,9 @@ function updateBrowsers(opts: Options) {
 }
 
 function getPlatformKey(browserName: BrowserName) {
-  const key = `${process.platform}:${browserName}`;
-  switch (key) {
-    case "darwin:firefox":
-    case "linux:firefox":
-    case "darwin:chromium":
-    case "linux:chromium":
-      return key;
+  const key = `${"windows"}:${browserName}`;
```

Comment:
I guess `"windows"` should be changed back to `process.platform`(?)

Reply:
thanks!